engine/src/main/java/org/camunda/bpm/engine/impl/incident/CompositeIncidentHandler.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership. Camunda licenses this file to you under the Apache License,
+ * Version 2.0; you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.camunda.bpm.engine.impl.incident;
+
+import org.camunda.bpm.engine.ProcessEngineException;
+import org.camunda.bpm.engine.impl.util.EnsureUtil;
+import org.camunda.bpm.engine.runtime.Incident;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+
+/**
+ * <p>
+ * An composite incident handler that handle incidents of a certain type by the multiple handlers.
+ * The result of handling depends on main handler.
+ * @see #mainIncidentHandler
+ * </p>
+ *
+ * @see IncidentHandler
+ *
+ * @author bespaltovyj
```

Comment:
According to our [code style](https://github.com/camunda/camunda-bpm-platform/wiki/Coding-Style-Java#java-keywords), author tags shouldn't be added. Can you remove this one?
```suggestion
```

Reply:
removed